Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -493,6 +507,42 @@ static void __GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameS
                 line->_width += glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index];
             }
 
+            DWRITE_GLYPH_METRICS glyphMetrics[run->_dwriteGlyphRun.glyphCount];
```

Comment:
Before we just got the maximum ascent and descent values from the font, which we could do here, but to be interoperable with the reference platform we need to only get the ascent and descent of the glyphs used in the line

---
In reply to: [80317746](https://github.com/Microsoft/WinObjC/pull/1027#discussion_r80317746) [](ancestors = 80317746,80313171)